packages/hardware-trezor/src/transformers/assets.ts

```diff
@@ -0,0 +1,51 @@
+import * as Trezor from 'trezor-connect';
+import { Cardano } from '@cardano-sdk/core';
+
+const compareAssetNameCanonically = (a: Trezor.CardanoToken, b: Trezor.CardanoToken) => {
+  if (a.assetNameBytes === b.assetNameBytes) {
+    return a.assetNameBytes > b.assetNameBytes ? 1 : -1;
+  } else if (a.assetNameBytes.length > b.assetNameBytes.length) return 1;
+  return -1;
+};
+
+const comparePolicyIdCanonically = (a: Trezor.CardanoAssetGroup, b: Trezor.CardanoAssetGroup) => {
+  if (a.policyId === b.policyId) {
+    return a.policyId > b.policyId ? 1 : -1;
```

Comment:
same here, this will always return -1 i think

Reply:
Done in: [1086d86](https://github.com/input-output-hk/cardano-js-sdk/pull/793/commits/1086d86fd46c35bc6b28ab2372b41e882893062f)